CHANGELOG.md

```diff
@@ -13,6 +13,19 @@ during a frame which would not run a simulation step.
 its default value is `TriMeshFlags::MERGE_DUPLICATE_VERTICES`,
 which was its hardcoded behaviour.
 
+### Modified
+
+- `RapierContext`, `RapierConfiguration` and `RenderToSimulationTime` are now a `Component`
+  - Rapier now supports multiple worlds, see example `multi_world3` for usage details.
+  - Migration guide:
+    - `ResMut<mut RapierContext>` -> `DefaultRapierContextAccessMut`
+    - `Res<RapierContext>` -> `DefaultRapierContextAccess`
```

Comment:
This is just a suggestion. Bevy has these `Read<>` and `Write<>` structs. Perhaps, to make the API look more idiomatic the access structs could be named as follow?
```suggestion
    - `ResMut<mut RapierContext>` -> `WriteDefaultRapierContext`
    - `Res<RapierContext>` -> `ReadDefaultRapierContext`
```

Reply:
TL;DR: ok :)

I tend to prefer suffixes to prefix, so when you're searching for a term, the alphabetical order comes better grouped.

That said, my initial naming does a bad job at putting `Default` first. For this reason, I might prefer a `RapierContextDefaultRead` ; *but that would probably mean renaming `DefaultRapierContext` to `RapierContextDefault`, which might sound weird.*

For the idiomatic way, I wanted to keep close to `Res` / `ResMut`, which is very idiomatic. I believe `Read`/`Write` is used for individual components, which is not exaclty the case here. I'm not sure I've seen it in `SystemParam`. But I'm not sure.

In any way, I don't feel too opinionated about that, so I'll just implement your suggestion :sweat_smile:.